apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -1421,7 +1423,7 @@ impl Webserver {
             show_message!(
                 MessageType::Highlight,
                 Message {
-                    action: "Next Steps".to_string(),
+                    action: "Next Steps  ".to_string(),
```

Comment:
are we removing the auto-padding?

Reply:
No - it is in there, but it is auto padding to the left. This is a weird case where we actually want the action field to look more centered